docs/source/index.rst

```diff
@@ -3,44 +3,55 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+=========================================
 Welcome to ``Flytekit``'s documentation!
 =========================================
 
+*************
+Flytekit API
+*************
+
 .. toctree::
    :maxdepth: 1
-   :caption: Flytekit Learn by Example
+   :caption: APIs
 
-   Flytekit Learn by Example <http://flytecookbook.readthedocs.io/>
-   Flyte Documentation <https://lyft.github.io/flyte/>
+   flytekit
+   testing
+   extend
+
+This section documents the various Python imports users will need to work with to build Flyte workflows, extend Flytekit, etc.
```

Comment:
nit: thoughts on moving this above the TOC? took me a moment to decipher which TOC it was referring to

Reply:
Actually i did it both ways.  I think in rendered form, it looks better here.